### DISCUSSION TITLE
admin examine - verb

### DISCUSSION THREAD

**Discussion by ArmoredArtist** [GoMudEngine/GoMud#121] [Ideas]
.: [•] The Frost Garden [Frostfang]
The Frost Garden is a mesmerizing blend of nature's resilience and ethereal
beauty. Amidst the icy realm of Frostfang, this garden thrives in quiet
defiance. Silver-blue ferns and frost-touched roses stretch towards the pale
sun, their petals and leaves glistening with delicate ice crystals. Towering
evergreens, draped in snow, stand sentinel, their boughs sheltering clusters of
winterberries and snowdrops. At the garden's heart lies a frozen pond, its
surface a mirror of glass, reflecting the dance of snowflakes and the occasional
aurora that paints the night sky. Winding pathways, carved through snowdrifts
and lined with luminescent moonflowers, guide visitors through this chilly yet
enchanting sanctuary. A small table and bench sit beside the pond.

In this room I want to interact with the table or the bench but I can not. As an admin logged in I want information on these items.
Is there a command to see all the objects in the room and their properties as an admin? if not, I suggest you make one.

Example "admin examine here" would bring up the "a small table" and "Bench" object if it was a real coded item in the game along with all its properties and commands to interact with it.

I am aware those two items might just be in the room description and not real objects but it is a decent idea for admins to check the properties of items while in game. Also it would allow me to tell if it is a real object vs a room description by stating what objects are in the room.

**Comment by Volte6** [comment]
I removed what would have made this possible when I incorporated scripting. I could tie it to scripting but I think that is way to much work for admins adding flavor to areas... 

I created an issue/task to re-add nouns to rooms and an admin command to do so here: https://github.com/Volte6/GoMud/issues/124

Closing this discussion to prevent duplicate discussion/tracking
